### DISCUSSION TITLE
基于Stun.Net我做了各个平台的发布

### DISCUSSION THREAD

**Discussion by snltty** [HMBSbige/NatTypeTester#244] [General]
基于Stun.Net我做了各个平台的发布
仅测试Nat类型功能
只有windows有UI，其它平台没有UI

https://github.com/snltty/ntt

**Comment by HMBSbige** [comment]
看样子还是用的 RFC3489（该协议已过时，不适用于现代路由器的 NAT 实现）

**Reply by snltty** [reply]
应该用5389吗

**Reply by HMBSbige** [reply]
> 应该用5389吗

当然了
